README.md

```diff
@@ -68,6 +68,24 @@ There are several command line arguments that can be used:
 * `-fromStep` : Optional argument which will allow you to start the sql-runner from any step in your playbook.
 * `-sqlroot`  : Optional argument to change where we look for the sql statements to run, defaults to the directory of your playbook.
 * `-var`      : Optional argument which allows you to pass a dictionary of key-value pairs which will be used to flesh out your templates.
+* `-consul`   : Optional argument which allows you to fetch playbooks and SQL files from a Consul server.
+* `-dryRun`   : Optional argument which allows you to run through your playbook without executing any SQL against your target(s)
+
+#### More on Consul
+
+Using the `-consul` argument results in the following changes:
+
+* The `-playbook` argument becomes the key that is used to look for the playbook in Consul.
+* The `-sqlroot` argument also becomes a key argument for Consul.
+
+Say for example you pass the following command in:
+
+* `./sql-runner -consul "localhost:8500" -playbook "sql-runner/playbook/1" -sqlroot CONSUL`
+
+This results in:
+
+* Looking for your playbook file at this key `sql-runner/playbook/1`
+* Expecting all your SQL file keys to begin with `sql-runner/playbook/1/<SQL path from playbook>`
```

Comment:
I added it as a way of having a valid default option when using Consul.  PLAYBOOK and BINARY both look up full paths whereas CONSUL simply keeps the -playbook argument without altering it at all.